### PR TITLE
added translation support with Polish translation

### PR DESCRIPTION
**Pull request by t-tomek**

### CODE REVIEW

File: BaseUpload.php [hunk 1]
```diff
@@ -38,6 +39,7 @@ class BaseUpload extends InputWidget
     public $clientEvents = [];
 
 
+    public $i18n = [];
```
Comment (tonydspaniard) [inline BaseUpload.php]: There are no phpDocs + it should have no space up and there should be two spaces between this attribute and the following function

File: BaseUpload.php [hunk 2]
```diff
@@ -52,4 +55,17 @@ public function init()
 
         $this->clientOptions['url'] = Url::to($this->url);
     }
+
```
Comment (tonydspaniard) [inline BaseUpload.php]: Needs phpDoc

**Comment by t-tomek** [comment]
done

**Comment by tonydspaniard** [comment]
@t-tomek Thank you!

**Pull request merged**